src/harness/unittests/transform.ts

```diff
@@ -138,6 +138,39 @@ namespace ts {
                 }
             };
         }
+
+        testBaseline("transformAwayExportStar", () => {
+            return ts.transpileModule("export * from './helper';", {
+                transformers: {
+                    before: [expandExportStar],
+                },
+                compilerOptions: {
+                    target: ts.ScriptTarget.ESNext,
+                    newLine: NewLineKind.CarriageReturnLineFeed,
+                }
+            }).outputText;
+
+            // Note: this is a simplified implementation that is missing some corner cases
+            // but is enough to show the bug.

```

Comment:
What bug? This comment has no reference to the original issue, so consider removing or adding details